packages/ketcher-react/src/script/editor/Editor.ts

```diff
@@ -559,9 +559,9 @@ class Editor implements KetcherEditor {
     }
 
     this.selection(null);
-    if (this._tool instanceof toolsMap.paste) {
-      this.event.change.dispatch();
-      return;
+
+    if (this._tool instanceof PasteTool) {
+      this._tool.updatePreview();
```

Comment:
could we at least unify logic, for ex. move updatePreview as empty method to Tool class and invoke this._tool.updatePreview() for all tools (remove if condition)? (implementation still be in PasteTool class)

Reply:
I did it like this, because I thought it was very specific to the paste tool, but your point makes sense too, I will change it.